tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/CouchbaseCommon.cs

```diff
@@ -124,5 +128,70 @@ internal static TOperationResult CommonOnMethodEnd<TOperationResult>(TOperationR
             state.Scope.DisposeWithException(exception ?? result.Exception);
             return tResult;
         }
+
+        internal static string GetNormalizedSeedNodesFromClientConfiguration(IClientConfiguration clientConfiguration)
+        {
+            if (ClientSourceToNormalizedSeedNodesMap.TryGetValue(clientConfiguration.Instance, out var normalizedSeedNodes))
+            {
+                return normalizedSeedNodes;
+            }
+
+            // Construct the normalized value from the list of hosts (each host will be {Host} or {Host}:{Port})
+            var sb = StringBuilderCache.Acquire(0);
+            IList<Uri> servers = clientConfiguration.Servers;
```

Comment:
`Servers` can never be null?

Reply:
No `Servers` cannot be null. It is mandatory, otherwise the client configuration will throw